Gems/Atom/RPI/Code/Include/Atom/RPI.Public/AssetTagBus.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Name/Name.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/std/containers/vector.h>
+#include <Atom/RPI.Public/AssetQuality.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        template<typename T>
+        class AssetTagInterface
+            : public AZ::EBusTraits
+        {
+        public:
+            using MutexType = AZStd::recursive_mutex;
+
+            virtual AssetQuality GetQuality(const AZ::Name& assetTag) const = 0;
+
+            virtual AZStd::vector<AZ::Name> GetTags() const = 0;
+
+            virtual void RegisterAsset(AZ::Name imageTag, const Data::AssetId& assetId) = 0;
+
+            virtual void RegisterTag(AZ::Name tag) = 0;
```

Comment:
Can you add comment for these functions? I couldn't tell what they do by reading the function name?
Also, do you need to change the input name 'imageTag' since the tag can be used for more than image asset?